api/controllers/itemController.js

```diff
@@ -1,21 +1,39 @@
-const { itemService } = require("../services");
-const { catchAsync } = require("../utils/error");
+const {itemService} = require("../services");
+const {catchAsync} = require("../utils/error");
+
+const getItems = catchAsync(async (req, res) => {
+    const items = await itemService.getItems(req.query);
+    res.status(200).json({items});
+})
+
+const registerItem = catchAsync(async (req, res) => {
+    const userId = req.user
+    const imageUrl = req.file.location
+    const result = await itemService.registerItem(userId, imageUrl, req.body);
+    if (result === false) {
+        res.status(200).json({message: "ARTIST_REGISTRATION_REQUIRED"})
+    } else {
+        res.status(200).json({message: "SUCCESSFULLY_REGISTERED", url: req.file.location});
+    }
```

Comment:
```suggestion
    if (!result) {
        return res.status(200).json({message: "ARTIST_REGISTRATION_REQUIRED"})
    }
    
    return res.status(200).json({message: "SUCCESSFULLY_REGISTERED", url: req.file.location});
```
1. 어차피 함수에서 bool 값을 반환하기 때문에 strict equality를 사용할 필요없이 부정연산자로 검증해보면 되겠습니다. 
2. 마찬가지로 if else 보다 위와 같이 작성되는 것이 가독성이 좋아보이고 지금같은 코드에서 return이 없으면 함수에서 로직이 종료되지 않고 다음으로 넘어가기 때문에 의도했던 것과 다르게 동작할 여지가 있습니다.